indexer/src/main.ts

```diff
@@ -88,8 +90,11 @@ export default async function transform({
         return null;
       }
 
```

Comment:
I think there is an issue on the couple of following lines you changed: you use `isTxExecuted` in order to filter out transaction that were executed, but that means you will never reach the end of the main for reverted transaction (line 188). 
IMO you would be better off:
- use the `includeReverted` filter
- check the `ExecutionStatus` field on the receipt
- if reverted, check the `revertReason` field
- if `revertReason` contains `out of resources -> place it in the `transactions_failure` collection

Reply:
messaged Fran in our common group to check with him about the `includeReverted`, let's wait for his answer